### PR TITLE
[CHI-2020] sponsorinfo and sponsor page

### PR DESCRIPTION
**Pull request by margaretvaltie**
updated sponsor page, sponsor info page for chicago 2020

### CODE REVIEW

File: content/events/2020-chicago/sponsor-info.md [hunk 1]
```diff
@@ -16,97 +16,36 @@ Email: [chicago-sponsors@devopsdays.org](mailto:chicago-sponsors@devopsdays.org)
 
 ### Important Dates
 * July 31: All payments due (no payment = no benefits)
-* August 10: Last day to order A/V equipment.
-* August 17: Last day to register for the event and get badge with name on it.
-* August 27: First day the venue will accept packages for the event.
-* September 1-2: DevOpsDays Chicago 2020!
+* August 17: Last day to order A/V equipment.
```
Comment (mattstratton) [inline content/events/2020-chicago/sponsor-info.md:19]: no longer relevant?

File: content/events/2020-chicago/sponsor-info.md [hunk 2]
```diff
@@ -16,97 +16,36 @@ Email: [chicago-sponsors@devopsdays.org](mailto:chicago-sponsors@devopsdays.org)
...
+* August 24: Gold 3 minute pre-recorded pitch video due
+* September 1: DevOpsDays Chicago 2020!
 
 ### Schedule
-The event takes place Tuesday and Wednesday, September 1-2, 2020. The detailed schedule for the two days can be found on the [DoDChi website](https://devopsdays.org/events/2020-chicago/program/). 
-NOTE: sponsors are able to get in early (7AM-9AM) September 1st to set up. Doors open at 8am and participants breakfast will be served at 8am in the sponsor area. 
+The event takes place Tuesday, September 1st, 2020. The detailed schedule for the two days can be found on the [DoDChi website](https://devopsdays.org/events/2020-chicago/program/).
```
Comment (mattstratton) [inline content/events/2020-chicago/sponsor-info.md:24]: this link does not exist yet fyi

**Comment by mattstratton** [comment]
I don't understand why Circle and Appveyor are having problems with this PR (they aren't even kicking off; something weird with the git stuff?) but if netlify can build it, i'll merge it past the failed checks.

@margaretvaltie I fixed a couple small things, but made two comments to review.

File: content/events/2020-chicago/sponsor-info.md (hunk 2 above)
Comment (margaretvaltie) [reply content/events/2020-chicago/sponsor-info.md:24]: I updated to 'will be posted' My plan is to not link to this before content is live (except for the prospectus, but nobody reads :)

Comment (mattstratton) [reply content/events/2020-chicago/sponsor-info.md:24]: It's also not the end of the world to have the link be a 404 (the beginnings of the schedule will probably be posted in a week or so anyway)

**Comment by mattstratton** [comment]
I pushed fixes for the commented items and this can be merged once deploy preview completes

**Pull request merged**